service/soa_service.go

```diff
@@ -0,0 +1,55 @@
+package service
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/insolvency-api/dao"
+	"github.com/companieshouse/insolvency-api/models"
+	"github.com/companieshouse/insolvency-api/utils"
+)
+
+// ValidateStatementOfAffairsRequest checks that the incoming statement of affairs request is valid
+func ValidateStatementOfAffairsRequest(statement models.StatementOfAffairs) string {
```

Comment:
I think the 2 functions in this file should be combined, this takes down the amount of code and checked that we have to do in the handler. We've done something similar in the `ValidatePractitionerDetails` method